### ISSUE TITLE
Presentation Deliverables for New Primitive Benchmarks

### ISSUE THREAD

**Issue by Michaeljurado24** [ezCGP/ezCGP#59]
Summary of What we Have Done

- Tan ran evolutionary runs without any new primitives so we can get a baseline.
- Ford added the flatten and the dense primitives for a separate run
- Henry added mutatable activations

What we need for the presentation
- We need the average final accuracy of the individuals from each run (running evaluator.py with 50 epochs)
- We need the t-statistics comparing Ford and Henry's run to the baseline run (Tan's run)
- We need to visualize the best individual using visualize.py

It would be good to have the raw results by Wednesday March the 3rd. Good luck team and thank you for your hardwork so far.

UPDATED INFORMATION ABOUT RUNS
-we are going to exclude the res_block in order to save time. That means comment it out from utils/training_block.py.
-If pace isn't working by Thursday we will use colab and ask Trai for help.

-We are going to change the number of evolutionary runs from 3 to 2 and increase the number of generations for each run. This means that afer the pbs script finishes we can run it again and it will continue from where it left off. 

To recap, each run has its own folder (i.e benchmark_runs/run1). For each run folder we are running the pbs script 2 times, one after the other. This means that if you see the "Wall Limit Reached" thing, just run the PBS script again Sam has code which starts the run back from where it left off. We want to do two runs, meaning that we need two run folders.

**Comment by hvl34179** [comment]
After 2 runs of evolutionary run, then change theses files to evaluate the outputs:
In problem.py:
SEED_ROOT_DIR = 'benchmark_runs/run_2' <- change it to which output folder you want to run.

in runEVAL.pbs:
# ezCGP pbs script
#PBS -N sam_200_epochs #<- change this to your job name 
#PBS -l nodes=1:ppn=8
#PBS -l pmem=14gb
#PBS -l walltime=12:00:00
#PBS -q pace-ice
#PBS -j oe
#PBS -o sam_200_epochs.out #<- change this to your output file name
#PBS -m abe
#PBS -M szhang460@gatech.edu #<- change this to your email

in evaluator.py
epochs = 30 
**Note: We only evaluate the output files for 30 epochs.
